docs/src/main/tut/dsl.md

```diff
@@ -68,7 +68,7 @@ and experiment directly in the REPL.
 ```tut
 val getRoot = Request[IO](Method.GET, uri("/"))
 
-val io = service.orNotFound.run(getRoot)
+val io = service.apply(getRoot).getOrElseF(NotFound())
```

Comment:
I think this was explained to me elsewhere, but I can't find the reference now.  What does the existing code do that's wrong?  I would think these are equivalent.

Reply:
At least in my case it seems to type check but once I run it, it cannot find orNotFound :/

Reply:
And now that I try it again, I naturally cannot reproduce... strange, wonder if I did a typo or if I hit some kind of corner case :S

Reply:
There needs to be an import.  In master, it's `org.http4s.implicits._`.  You might have been missing the import, or I think it also requires -Ypartial-unification.